app/views/messages/new.html.erb

```diff
@@ -1,24 +1,24 @@
-<div class="container-fluid" style="padding-top: 20px; padding-left:; width: 30%; float: centered;">
-  <div align="center">
-    <h1> Contact Form </h1>
-    <br>
-      <p>
-        Please send us feedback, comments, or any questions below.
-      </p>
-  </div>
-    <%= form_for @message, url: create_message_path do |f| %>
-
-      <%= @message.errors.full_messages.join(', ') %>
+<head>
+  <title>Contact</title>
+</head>
+<body class="bg-light">
+  <div class="contact-form container-fluid" style="padding-top: 20px; width: 500px">
+    <div class="py-5 text-center">
+      <h1>Contact Us</h1>
+      <br>
+      <p class="lead" style="font-size: 16px">Send us feedback, comments, or questions below, or email <a href="mailto:wiciberkeley@gmail.com" target="_top">wiciberkeley@gmail.com</a>.</p>
+    </div>
+      <%= bootstrap_form_for @message, url: create_message_path do |f| %>
 
-      <%= f.text_field :name, placeholder: 'Name', class: "form-control", style: "margin-top: 10px;" %>
+        <%= f.text_field :name, class: "form-control"%>
 
-      <%= f.email_field :email, placeholder: 'Email',class: "form-control", style: "margin-top: 10px;" %>
+        <%= f.email_field :email, placeholder: "you@example.com", class: "form-control" %>
 
-      <%= f.text_area :body, placeholder: 'Insert message here.', class: "form-control", style: "margin-top: 10px;" %>
+        <%= f.text_area :body, class: "form-control" %>
 
-      <%= f.submit 'Send', class: "btn btn-primary form-control", style: "margin-top: 10px;" %>
-    <% end %>
-    <p>
```

Comment:
make sure these static pages always take up the max height of the page, so the footer renders below them.

Reply:
I believe the default height is already 100%, but I'll add more styling to make sure.